tests/client_test.py

```diff
@@ -1169,6 +1170,48 @@ def test_get_price_history_every_minute_end_datetime(self):
             params=params)
 
 
+    @patch('tda.client.base.datetime.datetime', mockdatetime)
+    def test_get_price_history_every_minute_extendedhours(self):
+        self.client.get_price_history_every_minute(
+            'AAPL', need_extended_hours_data=True)
+        params = {
+                'apikey': API_KEY,
+                'periodType': 'day',
+                # ONE_DAY
+                'period': 1,
+                'frequencyType': 'minute',
+                # EVERY_MINUTE
+                'frequency': 1,
+                'startDate': MIN_TIMESTAMP_MILLIS,
+                'endDate': NOW_DATETIME_PLUS_SEVEN_DAYS_TIMESTAMP_MILLIS,
+                'needExtendedHoursData': True,
+        }
+        self.mock_session.get.assert_called_once_with(
+            self.make_url('/v1/marketdata/AAPL/pricehistory'),
+            params=params)
+
+
+    @patch('tda.client.base.datetime.datetime', mockdatetime)
+    def test_get_price_history_every_minute_no_extendedhours(self):
```

Comment:
I can't help but notice that this function is never actually called with the default parameters (i.e. `need_extended_hours_data=None`) This means that the most common path is actually never tested.

Reply:
Yup, makes sense to add default value to tests. 

The `needExtendedHoursData` should no longer appear in expected response params when passing `None`, and td ameritrade api will set to true by default.

We could also just initialize `need_extended_hours_data=True` when we pass it into utility functions, however, the current implementation works fine as is.

Latest commit includes the tests for `need_extended_hours_data=None` scenarios.